cmd/project_list.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/xanzy/go-gitlab"
+	lab "github.com/zaquestion/lab/internal/gitlab"
+)
+
+var projectListCmd = &cobra.Command{
+	Use:     "list [page]",
+	Aliases: []string{"ls"},
+	Short:   "List all projects",
+	Run: func(cmd *cobra.Command, args []string) {
+		page := 0
+		if len(args) == 1 {
```

Comment:
Can probably use the `parseArgs` function defined in `cmd/root.go` to accept a search and/or page. Similar to how other commands accept a remote and/or page.

Reply:
I beleive ListProjectOptions can accept such a search parameter, but I may be mistaken.

Reply:
This is optional, was just thinking out loud here.